Reason: Fix generated config to be in line with the stock perl one
Upstream: submitted, https://developer.berlios.de/patch/index.php?func=detailpatch&patch_id=3549&group_id=12304

From b97f89b71c2a4b0c9d5b67e6102d1dec676f7475 Mon Sep 17 00:00:00 2001
From: Benedikt Morbach <benedikt.morbach@googlemail.com>
Date: Sun, 29 Sep 2013 00:24:57 +0200
Subject: [PATCH] there is no such thing as {site,vendor}scriptdir
Organization: Exherbo

---
 cnf/configure_genc.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cnf/configure_genc.sh b/cnf/configure_genc.sh
index 062eeb2..f3eb1f5 100644
--- a/cnf/configure_genc.sh
+++ b/cnf/configure_genc.sh
@@ -151,7 +151,7 @@ default_inst sitehtml3dir
 default_inst sitelib 
 default_inst siteman1dir
 default_inst siteman3dir
-default_inst sitescriptdir sitescript
+default_inst sitescript
 default_inst vendorarch
 default_inst vendorbin
 default_inst vendorhtml1dir
@@ -159,7 +159,7 @@ default_inst vendorhtml3dir
 default_inst vendorlib
 default_inst vendorman1dir
 default_inst vendorman3dir
-default_inst vendorscriptdir vendorscript
+default_inst vendorscript
 default installstyle lib/perl5
 default installusrbinperl define
 
-- 
1.8.4

